actionmailbox/test/test_helper.rb

```diff
@@ -54,3 +54,5 @@ def bounce(to:)
     end
   end
 end
+
+require_relative "../tools/test_report" if ENV["CI"]
```

Comment:
Why is this just `CI`?

Reply:
I'm a little concerned about loading files outside of packages in tests that normally use. 
I think it would be better to check the existence of `tools/test_report` or make the env name a little more unique.

Reply:
> Why is this just CI?

Just my mistake 🙇